src/ConsoleCommand.php

```diff
@@ -18,6 +19,7 @@ public function __construct(
         /** @var string[] */
         public readonly array $aliases = [],
         public readonly ?string $help = null,
+        public readonly ?CronDefinition $cron = null,
```

Comment:
for convenience this could accept either crondefinition, a cron string or dateinterval and then normalize it to CronDefinition

Reply:
What I envisioned is only one way to avoid confusion. You can read some more about my vision on this topic here: https://stitcher.io/blog/opinion-driven-design